app/src/lib/stores/notifications-store.ts

```diff
@@ -283,11 +290,11 @@ export class NotificationsStore {
       return
     }
 
-    // Ignore any remaining notification for check suites that started along
+    // Ignore any remaining notification for check runs that started along
     // with this one.
     for (const check of checks) {
       if (check.checkSuiteId !== null) {
```

Comment:
Should this `check.checkSuiteId !== null` be changed to `check.checkSuiteId === event.check_suite_id`?

Reply:
Nope, because we're trying go skip notifications from _other_ check suites. However, maybe we can even remove that `if` and just add all `check.id` to the set 🤔

Reply:
Done in 5d4fa4f2bbbecbdeb4c60a61bde3ee7fb39b567b !